File: April/Abiji-2020/Golang /Length of Last Word In string/Length of Last Word In string.go
func lengthOfLastWord(s string) int {
    var lens int =0
    var prev int =0
    for _, v := range s{
        if v == ' '{
            lens =0
            continue;
        } else {
            lens++;
        }
        prev = lens
    }
    if lens == 0 {
        return prev
    }
    return lens
}

File: April/Abiji-2020/C++/Length of Last Word In string/Length of Last Word In String.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int result =0;
        int prev =0;
        for(int i=0;s[i]!='\0';i++){
            if (s[i] == ' '){
                result = 0;
                continue;
            } else {
                result++;
            }
            prev = result;
        }
        return result==0? prev: result;
        
    }
};
